src/ui/server/controllers/insurance/account/sign-in/index.ts

```diff
@@ -83,6 +84,23 @@ export const post = (req: Request, res: Response) => {
     });
   }
 
-  // TODO: check sign in details are correct and generate a code.
-  return res.redirect(ENTER_CODE);
+  try {
+    // validate credentials
```

Comment:
Can we please add `req.body` void check.

Reply:
This is done by TS, a `req` will always have a body